arm_config:
  # pattern is: name:ip_addr:port:offset(rad)
  - arm_joint_0:10.0.0.121:9081:0
  - arm_joint_1:10.0.0.121:9082:0
  - arm_joint_2:10.0.0.121:9083:0
  - arm_joint_3:10.0.0.121:9084:0
